src/Autowhatever.js

```diff
@@ -431,7 +431,7 @@ export default class Autowhatever extends Component {
     });
 
     return (
-      <div {...containerProps}>
+      <div aria-controls={itemsContainerId} {...containerProps}>
```

Comment:
I don't necessarily have an issue with the `aria-controls` attribute, but what was the rationale behind adding it explicitly as opposed to configuring it in the `containerProps` that we can control via props?

Reply:
I think it should always match the `aria-controls` attribute that is set on the `input` as well, (set on `inputComponent` above this in the file) and since the user can't control that one, I figured this should be set explicitly as well.

Reply:
BUT, looking at it again, I should just add it to the `containerProps` object that is created above this. One sec.

Reply:
AND now I want to see if I can just replace `aria-owns` (which is set to the same id) with this `aria-controls`.

Reply:
Ok, that looks like the key change!

Reply:
👍